util/venv-dependencies.sh

```diff
@@ -199,7 +211,8 @@ fi
 # In case we have a new virtualenv executable.
 hash -r
 
-if ! virtualenv --python=python2 $virtualenv_no_site "$1"; then
+if ! virtualenv --python=python2 $NO_SITE_PACKAGES "$1"; then
+    dn="$(dirname "$1")"
```

Comment:
What is the `dn="$(dirname "$1")"` change here doing? I'm likely missing something but maybe that's a sign that there needs to be a comment here.